package.json

```diff
@@ -31,7 +31,7 @@
   },
   "dependencies": {
     "@popperjs/core": "^2.4.4",
-    "@symphony-ui/uitoolkit-styles": "1.6.0-rc.1",
+    "@symphony-ui/uitoolkit-styles": "^2.0.0",
```

Comment:
Is "^" on purpose?

Reply:
Yes, in my point of view, we should no 'lock' on a specific version.

Reply:
In our case, we release TK-Components and TK-Styles, so there is no a really impact. But, it's not a good practice to lock on a specific version in the package.json. Because you can prevent those who use the library to get some fixes (released in your dependencies).

Fox example: 
Let 3 libs: A => B => C. (A uses B which uses C)

Imagine we locked the version of C (v2.0.0) in the package.json of B. Next, C provide a security fix (v2.0.1).
Then, A will not be able to get the v2.0.1 of C when it installs B.
Whereas, if the version is not locked, then if 'A' install B, it will get C(v2.0.1)